common/views/pages/_app.js

```diff
@@ -450,8 +449,20 @@ export default class WecoApp extends App {
             src="https://i.wellcomecollection.org/assets/libs/picturefill.min.js"
             async
           />
-          <JsonLd data={museumLd(wellcomeCollectionGalleryWithHours)} />
-          <JsonLd data={libraryLd(wellcomeLibraryWithHours)} />
+          <script
```

Comment:
We had to remove this due to:

> title, meta or any other elements (e.g. script) need to be contained as direct children of the Head element, or wrapped into maximum one level of <React.Fragment> or arrays—otherwise the tags won't be correctly picked up on client-side navigations.

-- https://nextjs.org/docs/api-reference/next/head

Reply:
would `JsonLd` not still resolve to a script tag that's a direct child of the head?

Reply:
It does, but it is still considered by JSX as nested.